src/main/java/frc/robot/subsystems/manipulator/ManipulatorIO.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.subsystems.manipulator;
+
+import frc.robot.subsystems.flywheel.FlywheelIO;
+import org.littletonrobotics.junction.AutoLog;
+
+public interface ManipulatorIO {
+    @AutoLog
+    public static class ManipulatorIOInputs {
+        public double positionRad = 0.0;
+        public double velocityRadPerSec = 0.0;
+        public double appliedVolts = 0.0;
+        public double[] currentAmps = new double[] {0.0};
+    }
+
+    /** Updates the set of loggable inputs. */
+    public default void updateInputs(ManipulatorIO.ManipulatorIOInputs inputs) {
+    }
+
+    /** Run closed loop at the specified velocity. */
+    public default void setVelocity(double velocityRadPerSec, double ffVolts) {
+    }
+
+    /** Stop in open loop. */
+    public default void stop() {
+    }
+
+    /** Set velocity PID constants. */
+    public default void configurePID(double kP, double kI, double kD) {
+    }
```

Comment:
Pid on the motor is something we don't use. When you send a configurePID command to the motor, you are telling it to do a pid loop for velocity directly on the motor controller. This is not bad but we just do not do it on DARC SIDE.
```suggestion
```